tests/asyncio/gapic/test_method_async.py

```diff
@@ -198,41 +198,6 @@ async def test_wrap_method_with_overriding_retry_and_timeout(unused_sleep):
     method.assert_called_with(timeout=22, metadata=mock.ANY)
 
 
-@mock.patch("asyncio.sleep")
-@mock.patch(
-    "google.api_core.datetime_helpers.utcnow",
-    side_effect=_utcnow_monotonic(),
-    autospec=True,
-)
-@pytest.mark.asyncio
-async def test_wrap_method_with_overriding_retry_deadline(utcnow, unused_sleep):
```

Comment:
Why was this removed ?

Reply:
Just updated the PR description. The last few sentences of the item 8 describe the removal of unnecessary logic, and this test was testing it. From the PR description for your convenience

> Also essentially rollback the https://github.com/googleapis/python-api-core/commit/52f12af027568ddeae7fb6b1e3d2111cce1bac08 change, since that is inconsistent with the other languages and damages uniform distibution of retry delays artificially shifting their concentration towards the end of timeout.